docs/content/configuration/upstream.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Configuring Upstream Proxy to use Go modules repository
+description: How to Fetch modules directly from a Go modules repository such as GoCenter
+weight: 1
+---
+
+The upstream proxy used when fetching modules directly is by default the actual source, e.g github.com. This can be configured to use a Go modules repository like GoCenter.
```

Comment:
Since the upstream is really a VCS (e.g. github.com like you wrote), can we reword this sentence to indicate that? Here's one idea that you could build on if you want:

"By default, Athens fetches module code from an upstream VCS like github.com, but..."